test/helpers/deployer.js

```diff
@@ -163,6 +163,14 @@ class VotingDeployer {
     return token
   }
 
+  async setOracle(address) {
```

Comment:
Maybe rename this to something like "grantOraclePermissionToAddress". setOracle is kind of ambiguous.